core/templates/dev/head/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts

```diff
@@ -760,6 +765,33 @@ oppia.directive('conversationSkin', [
                 var nodeId = UrlService.getUrlParams().node_id;
                 StoryViewerBackendApiService.recordStoryNodeCompletion(
                   storyId, nodeId);
+
+                StoryViewerBackendApiService.fetchStoryData(storyId).then(
+                  function(storyDataDict) {
+                    var storyNodes = storyDataDict.story_nodes.map(
+                      function(storyNodeDict) {
+                        return ReadOnlyStoryNodeObjectFactory
+                          .createFromBackendDict(storyNodeDict);
+                      });
+                    var completedStoryNodes = [];
+                    storyNodes.forEach(function(storyNode) {
+                      if (storyNode.isCompleted) {
+                        completedStoryNodes.push(storyNode);
+                      }
+                    });
+                    if (completedStoryNodes.length %
+                      NUM_EXPLORATIONS_PER_REVIEW_TEST === 0 ||
```

Comment:
Just a design question over here... Shouldn't the creator of the story have control over this parameter? I think for now, while we lean towards curated lessons it should be ok to have this as a constant. But just wanted to know if this has been discussed before.

Reply:
We haven't really discussed this, but I think it is OK for this to be hardcoded. If there is a need to configure this later on, we can easily do that as a feature request.